opm/simulators/flow/FlowGenericVanguard.cpp

```diff
@@ -325,11 +326,9 @@ void FlowGenericVanguard::init()
     // Initialize parallelWells with all local wells
     const auto& schedule_wells = schedule().getWellsatEnd();
     parallelWells_.reserve(schedule_wells.size());
-
-    for (const auto& well: schedule_wells)
-    {
-        parallelWells_.emplace_back(well.name(), true);
-    }
+    std::transform(schedule_wells.begin(), schedule_wells.end(),
+                   std::back_inserter(parallelWells_),
+                   [](const auto& well) { return std::make_pair(well.name(), true); });
```

Comment:
This replaces in-place construction with construct+move.  Is that really what we want to do?

Reply:
Also, the old code is more readable. If static analysis is complaining about this, maybe we could tune the analysis a bit?

Reply:
That is in the eye of the beholder. We've had this discussion before..

I was on the fence whether or not to sacrifice in-place, I can remove this one.